src/main/java/frc/robot/subsystems/PoseEstimator.java

```diff
@@ -97,4 +98,12 @@ public void resetOdometry(){
     public void resetOdomGivenPose2d(Pose2d pose) {
 		swerveDrivePoseEstimator.resetPosition(pigeon.getYawAsRotation2d(), swerveSubsystem.getPositions(), pose);
 	  }
+
+    public double getPoseX(){
+        return swerveDrivePoseEstimator.getEstimatedPosition().getTranslation().getX();
+    }
+
+    public double getPoseY(){
+        return swerveDrivePoseEstimator.getEstimatedPosition().getTranslation().getX();
+    }
```

Comment:
Why are getPoseX() and getPoseY() returning the same thing? If you don't need this, remove it. Otherwise, change it.